llvm/test/CodeGen/AMDGPU/move-to-valu-lshl_add.ll

```diff
@@ -0,0 +1,90 @@
+; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 6
+; RUN: llc -mtriple=amdgcn -mcpu=gfx1200 -stop-after=si-fix-sgpr-copies < %s | FileCheck %s
+
+define amdgpu_kernel void @lshl1_add(ptr addrspace(5) %alloca) {
```

Comment:
Not an alloca (and it's likely this may be made a verifier error some day), so can you use a different pointer